fcc-test-suite-template-master/index.html

```diff
@@ -60,6 +60,34 @@ <h1 id="title">Tito and Danny Bazar</h1>
          <button id="submit" type="submit" class="txts">Submit Feedback</button>
        </div>
      </form>
+     <ul class="projects">
+       <li class="project">
+         <img class="image" src="./img/snapshot.png" alt="">
```

Comment:
I would recommend that you add an src to your img elements. This helps with screen readers and also SEO